changelogs/fragments/2370-zos_replace-1-4-0-compatiablity.yml

```diff
@@ -0,0 +1,5 @@
+minor_changes:
```

Comment:
Since this is to support ZOAU 1.4.0 this time this will have to be trivial as is something not customer facing, at least at module level. We will announce 1.4.0 support as an overall collection improvement.